lib/drivers/medtronicDriver.js

```diff
@@ -68,7 +68,26 @@ module.exports = function (config) {
   };
 
   var MESSAGES = {
-    READ_HISTORY : 0x80
+    READ_SETTINGS : 0xC0,
+    READ_HISTORY : 0x80,
+    READ_CARB_RATIOS : 0x8A,
+    READ_INSULIN_SENSITIVITIES: 0x8B,
+    READ_MODEL : 0x8D,
+    READ_PROFILE_STD : 0x92,
+    READ_PROFILE_A : 0x93,
+    READ_PROFILE_B : 0x94,
+    READ_CURRENT_PAGE : 0X9D,
+    READ_BG_TARGETS : 0x9F
+  };
+
+  var BG_UNITS = {
+    1: 'mg/dL',
+    2: 'mmol/L'
+  };
+
+  var CARB_UNITS = {
+    1 : 'grams',
+    2: 'exchanges'
```

Comment:
So we're supporting this? Should be added to the checklist somewhere if so. Maybe in a device-specific info checklist section for now since I'm not sure any other device support exchanges and it would be tedious to update all of them...